File: BudgetSource/CSharpFunction/Internal/InternalTestbed.cs
namespace CSharpFunction.Internal
{
    using InputModel = Schema.SampleSchemas.TextDocument;
    using OutputModel = Schema.SampleSchemas.AllCaps;
    public class InternalTestbed
    {
        public OutputModel HandleData(InputModel data)
        {
            var output = new OutputModel();
            output.originalcontent = data.content;
            output.allcapscontent = data.content.ToUpper();
            output.transformedcount = data.content.Where(c => char.IsLower(c)).Count();
            return output;
        }
    }
}

File: BudgetSource/PythonFunction/internal/internal_testbed.py
from internal.schema.sample_schemas import TextDocument, AllCaps


class InternalTestbed:
    def handle_data(self, data: TextDocument) -> AllCaps:
        output = AllCaps()
        output.originalcontent = data.content
        output.allcapscontent = data.content.upper()
        output.transformedcount = sum(1 for c in data.content if c.islower())
        return output
